src/app/review-item/components/review-page/review-page.component.ts

```diff
@@ -65,11 +65,15 @@ export class ReviewPageComponent implements OnInit, OnDestroy {
     }
   ];
 
-  description =
+  description_intro =
     'Im Tatbestand unten siehst du alle für den Fall relevanten Informationen. Mache dich zunächst mit dem ' +
     'Fall vertraut und überlege, ob du ihn wirklich bearbeiten willst. Wenn du dich dazu entschlossen hast, ' +
     'klicke bitte auf "Fall bearbeiten" und starte Deine Bewertung.';
 
+  description_review =
```

Comment:
@Th3da Kannst du bitte über den alternativen Text schauen für die Review-Page nachdem ein Fall angenommen wurde. Wenn du daran etwas ändern willst, dann mach das gerne direkt auf dem Branch hier.

Reply:
Prima soweit! 
Habe nur kleine Änderungen an beiden Stellen vorgenommen.

Unter den oben beschriebenen Test-Bedingungen (items.service.ts) funktioniert der Hinweis (no open cases).
Wenn ich aber tatsächlich meinen letzten Fall bearbeitet habe, funktioniert es nicht. Liegt das daran, dass noch Änderungen in den Lambdas waren (da seh ich aber keinen offenen PR)?

Reply:
Ah cool, danke fürs Testen! Ich hatte keinen User, der alle Fälle durch hatte. 
Kannst du es jetzt nochmal testen, hab im Backend gerade noch ne kleine Änderung eingebaut.

Reply:
! funktioniert !

Reply:
Was hältst Du vom Vorschlag oben (no open cases)?
Würde dann einen Commit auf diesen Branch machen ... ?

Reply:
Hab meine Änderungen jetzt auf den Branch gepusht ...